README.md

```diff
@@ -11,43 +11,46 @@ To setup any example, follow the below steps.
 
 2. Find the `.vf` file in `/sample`, e.g, the `hello-world` folder has a `hello-world.vf` file. This file contains a ready-made Voiceflow project which will be integrated with the JavaScript code.
 
-2. Upload the `.vf` file to [Voiceflow](https://creator.voiceflow.com) to import the project. See [here](https://docs.voiceflow.com/#/platform/project-creation/project-creation?id=project-creation) and click the "Import a .vf file" tab for instructions on how to do this.
+3. Upload the `.vf` file to [Voiceflow](https://creator.voiceflow.com) to import the project. See [here](https://docs.voiceflow.com/#/platform/project-creation/project-creation?id=project-creation) and click the "Import a .vf file" tab for instructions on how to do this.
 
-3. Open the imported project on Voiceflow
+4. Click on the `Manage API Keys` button and generate an API key for your workspace. You will need this API key for later.
```

Comment:
I couldn't find where Manage API Keys was suppose to be. It would be helpful to:
1. Tell the user to navigate to another screen if necessary. If the users needs to be at a different screen than the Dashboard (where they imported the project in the last step), then we should provide instructions to do this.
2. Include an image highlighting the thing to click if it's not obvious (see the images below as an example).